RELEASE.md

```diff
@@ -12,43 +12,47 @@ Make a PR which commits the following:
    -version = "0.1.0"
    +version = "1.2.3"
    ```
-   
+
 2. Commits updated API docs
 
    ```sh
    (.venv) bytewax/apidocs $ pip install -r requirements.txt
    (.venv) bytewax/apidocs $ rm -rf html/
    (.venv) bytewax/apidocs $ ./build.sh
    ```
-   
+
    You'll get a warning about `Couldn't read PEP-224 variable
    docstrings`, but ignore that as our PyO3 Rust pyclasses don't have
    Python source `pdoc` can read.
-   
+
    If we've been committing these as we go, there might not be any
    changes to commit here. That's fine.
-   
+
 3. Labels the latest changelog entries with the version number
 
    Look in `CHANGELOG.md` for the latest batch of hand-written
    changelog notes and add a new headings with the version number.
-   
+
    ```diff
    *** CHANGELOG.md
     ## Latest
-    
+
     __Add any extra change notes here and we'll put them in the release
     notes on GitHub when we make a new release.__
-   
+
    +## 1.2.3
    +
     * Example note here. Describe any super important changes that you
       wouldn't glean from PR names which will be added by GitHub
       automatically.
    ```
-   
+
 Approve and merge that PR.
 
+4. Check that the CI run completed for the just updated `main` branch
```

Comment:
@davidselassie do you think we should add a loop waiting 20 or 30 minutes retrying in the CD workflow?

we can do that